egs2/TEMPLATE/asr1/asr.sh

```diff
@@ -335,6 +335,19 @@ if [ ${skip_stage} -ne 2 ] && [ ${stage} -le 2 ] && [ ${stop_stage} -ge 2 ]; the
         log "${feats_type} is not supported yet."
         exit 1
 
+    elif  [ "${feats_type}" = extracted ]; then
+        log "Stage 2: ${feats_type} extract: data/ -> ${data_feats}/"
+
+        for dset in "${train_set}" "${dev_set}" ${eval_sets}; do
+            mkdir -p ${data_feats}/${dset}
+            cp -r data/${dset}/* ${data_feats}/${dset}
```

Comment:
It's better to avoid to use cp for data directory because sometimes the directory contains features matrix data. 

Hmm, but copy_data_dir.sh requires wav.scp, as you said... I'll let you know if I got better idea.

Reply:
I guess we could add a "custom_copy_data_dir_sh" with an `exclude-files` option to `espnet/utils`? But I don't know if it would be useful outside how2?

Reply:
> Yes if I can figure out the problem with my repo, I can't seem to create a new branch synced with espnet:v.0.7.0 ...

Well, is this simply a problem of git usage?

```bash
git remote add origin https://github.com/espnet/espnet/
git checkout -b <branch name> origin/v.0.7.0
```


By the way, when creating PR, it's better to create a new branch to let the base branch to track the original branch. This is just a github's manner.

Reply:
I'm not sure which is better, but for now can you add the following?

```bash
<"${data_feats}"/feats.scp awk ' { print($1,"<DUMMY>") }' > "${data_feats}"/wav.scp
utils/copy_data_dir.sh data/"${dset}"/* "${data_feats}/${dset}"
```